kahuna/public/js/components/gr-tooltip/gr-tooltip.js

```diff
@@ -20,6 +18,16 @@ tooltip.directive('grTooltip', ['onValChange', function (onValChange) {
                     element.attr('data-title', newTooltip);
                 }));
             }
+
+            element.bind('mouseenter', () => {
+                element.addClass('titip-default')
```

Comment:
maybe the second `.addClass` should be directly below the first, rather than starting at the last `e` of `element` (same for `removeClass` below?